creator-cards/calvin-allen.json

```diff
@@ -27,5 +31,9 @@
       "type": "LinkTagRssType",
```

Comment:
Hey @CalvinAllen we changed the feed's a while ago. 

Please use as type "dotnet" or "personal" depending on if the feed only contains .NET and related articles or use  personal if it contains other things as well.

Reply:
Ah, alright, I'll get it updated. I think I might spin out a dotnet specific feed, too, even though 99% of the time thats all I post about

Reply:
@DeeJayTC Should be good to go now